plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/ext/LocalStorageDockerVolumePathProvider.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.docker.machine.ext;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Provider;
+
+/**
+ * Provides volumes configuration of machine for terminal
```

Comment:
My meaning is that the code should be more documented for someone that want to read it.

public ContainerCreated(String id, String[] warnings) { <-- what are these warnings

public static final String API_ENDPOINT_URL_VARIABLE = "CHE_API_ENDPOINT";
what is the purpose of this constant : why the name is different of the content ?

LocalStorageDockerVolumePathProvider : wrong documentation

Also on a separated note, why servicetest is modified ? 
what is the relation of the issue with that change ? it seems to be a cleanup and a change from tmp to fake. Looking at the history we may wonder what the relation of this change and the name of the commit

also as there was a commit related to this PR, why status is not merged  and closed ?